cli/smartcontract/smart_contract.go

```diff
@@ -337,6 +337,21 @@ func NewCommands() []cli.Command {
 					},
 				},
 			},
+			{
+				Name:   "calc-hash",
+				Usage:  "calculates hash of a contract after deployment",
```

Comment:
`before` deployment?

Reply:
It's calculating what the hash will be after deployment before deployment, yeah.